cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -148,7 +148,11 @@ def _is_migration_started(vm):
         wait_for(func=_is_migration_started, func_args=[vm],
             message="migration has not started for all VMs", delay=5, num_sec=300)
 
-    host_creds = provider.hosts.all()
+    if provider.one_of(OpenStackProvider):
+        host_creds = provider.appliance.collections.openstack_nodes.all()
```

Comment:
Its LGTM but I think you are searching for all host/nodes
what you thing about
```
appliance.collections.hosts.all()
```

Reply:
@digitronik  We are looking for provider specific host only.